src/colors.tsx

```diff
@@ -16,6 +16,7 @@
 export const background = '#F6F7F9';
 export const primary = '#681DA8'; // Google Purple 900.
 export const secondary = 'rgba(0, 0, 0, .54)';
+export const ternary = 'rgba(0, 0, 0, .38)';
```

Comment:
1. The contrast of this color is really bad -- is there a reason why we have to use this color for comments? I have concerns about accessibility. I don't see how this color can be safely used elsewhere either.

If we were to keep this: the name seems to imply it is like `primary` and `secondary` and may get confused with MDC theming. Thoughts on limiting it to component-scope for now and revisit?

Reply:
I also see that I made a typo on the name of the constant, I meant to write `tertiary`, not `ternary`

I changed the color of comments to a light gray because Marissa suggested that change.

![image](https://user-images.githubusercontent.com/34299428/105770660-20950d80-5f25-11eb-9c40-3ae14d148d6a.png)

She actually suggested using the `textBlackSecondary`, but I changed it to `textBlackTertiary` because I thought the secondary text color did not have enough contrast with the color of the code.

![image](https://user-images.githubusercontent.com/34299428/105770891-681b9980-5f25-11eb-8f31-a6549eaaeb8d.png)

Should we change the color? I see that the console uses a very contrasting red color for comments, we could maybe change it to red if we want to keep the color consistency.

![image](https://user-images.githubusercontent.com/34299428/105771126-b4ff7000-5f25-11eb-8d5d-bd1b3265de59.png)

Reply:
Also, this color is used by the `InspectionSection` component in the next PR, and since it is an official firebase spec color, I think it could live as a constant in the `colors.tsx` file, changing the name to maybe `textBlackTertiary` could help avoiding any confusion.

Reply:
Let's keep it as `textBlackTertiary` for now and since we're using it in `InspectionSection` I'd say it can stay. We can revisit this later.